config/job.yaml.tim

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sd-
```

Comment:
Let's not use generate name.  We want to tag this with the `buildId` so we can easily look it up later.  (note we can use tags for this as well)